src/js/directives/image-list-directive.test.js

```diff
@@ -17,14 +17,16 @@ describe('Image List Directive', function () {
   beforeEach(inject(function (_$rootScope_, _$compile_, _$log_) {
     $rootScope = _$rootScope_;
     $compile = _$compile_;
+
+
```

Comment:
Remove one of these lines. Should only have a single line break between code blocks.